From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Sat, 19 Mar 2016 19:02:15 +0100
Subject: [PATCH] FIXME: Disable amd64 Liberty on Ubuntu for now.

Until we fix all cohabitation of amd64 and arm64, disable Openstack
release Liberty on Ubuntu (x86_64) in openstack.yaml fixture.

TODO: maybe rename above fixture to include arch in name.
---
 nailgun/nailgun/fixtures/openstack.yaml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/nailgun/nailgun/fixtures/openstack.yaml b/nailgun/nailgun/fixtures/openstack.yaml
index 4979fb5..037ceba 100644
--- a/nailgun/nailgun/fixtures/openstack.yaml
+++ b/nailgun/nailgun/fixtures/openstack.yaml
@@ -1986,6 +1986,7 @@
   extend: *base_release
   fields:
     name: "Liberty on Ubuntu 14.04"
+    state: "unavailable"
     version: "liberty-8.0"
     arch: "amd64"
     can_update_from_versions: []
